lib/ueberauth/strategy/keycloak/oauth.ex

```diff
@@ -29,20 +30,21 @@ defmodule Ueberauth.Strategy.Keycloak.OAuth do
   These options are only useful for usage outside the normal callback phase of Ueberauth.
   """
   def client(opts \\ []) do
-    config =
-      :ueberauth
-      |> Application.fetch_env!(Ueberauth.Strategy.Keycloak.OAuth)
-      |> check_config_key_exists(:client_id)
-      |> check_config_key_exists(:client_secret)
-
     client_opts =
       @defaults
-      |> Keyword.merge(config)
+      |> Keyword.merge(config())
       |> Keyword.merge(opts)
 
     OAuth2.Client.new(client_opts)
   end
 
+  defp config() do
```

Comment:
Can you add a short docstring on what this function does (e.g. it would be useful to say where it is grabbing the config from)